relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -490,6 +534,42 @@ fn timestamp_by_op(spans: &[Annotated<Span>], op: &str) -> Option<Timestamp> {
         .and_then(|span| span.timestamp.value().copied())
 }
 
+/// Counts the number of dynamically loaded images from debug meta.
+fn count_images(debug_meta: &Annotated<DebugMeta>) -> Option<f64> {
+    debug_meta
+        .value()
+        .and_then(|debug_meta| debug_meta.images.value())
+        .map(|images| images.len() as f64)
+}
+
+/// Sum the sizes of dynamically loaded images from debug meta.
+fn sum_images_size(debug_meta: &Annotated<DebugMeta>) -> Option<f64> {
```

Comment:
nit: I would convert to f64 at the latest possible point, i.e. return `Option<u64>` here. Unless we expect values so big that integer addition overflows.